source/content/tax-exempt-status.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Request Tax Exempt Status
+description: Provide this information to file your qualified tax exempt status with Pantheon.
+tags: [billing]
+categories: [manage]
+contributors: [edwardangert]
+searchboost: 150
+reviewed: "2020-03-24"
+---
+
+To make it easy for our team to confirm your tax-exempt status and have it reflected on your bill quickly, [contact Support](/support) with the following information:
+
+- Copy of tax exemption determination letter
```

Comment:
Does this mean they need to have a copy handy, or they need to send a copy to Support? I'm guessing the latter, and that this should be clarified to say so.

Also, does it need to be in PDF format? Does a cell phone picture of the letter suffice? You know someone's gonna try that.